lib/rubocop/ast/node.rb

```diff
@@ -356,6 +368,10 @@ def empty_source?
         {assignment? (send _recv :<< ...)}
       PATTERN
 
+      def enumerating?(method)
```

Comment:
@koic @Drenmi Do you think this is going to be useful in other cops or should we push it in the only cop using it?

Reply:
I don't know offhand 💦 

> push it in the only cop using it?

In that case, I like this choice because it's YAGNI :-)